target/linux/ipq40xx/files-6.6/arch/arm/boot/dts/qcom/qcom-ipq4018-ap4220.dts

```diff
@@ -0,0 +1,391 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+
+/ {
+	compatible = "alibaba,ap4220", "qcom,ipq4019";
+	model = "Alibaba AP4220";
+
+	aliases {
+		ethernet0 = &gmac;
```

Comment:
Considering you are setting the MAC anyway it literally makes no sense to have this alias nor set the label MAC to it